src/components/MapView/Legends/index.tsx

```diff
@@ -27,14 +28,26 @@ function Legends({ classes, layers }: LegendsProps) {
   const isAnalysisLayerActive = useSelector(isAnalysisLayerActiveSelector);
 
   const legendItems = [
-    ...layers.map(({ title, legend, legendText }) => {
-      if (!legend || !legendText) {
+    ...layers.map(layer => {
```

Comment:
Maybe deconstruct baseUrl, serverLayerName, and type here?

Reply:
I tried previously. However, Typescript validates the deconstruction of a LayerType interface instead of a WMSLayer one. This is why I make the check after this